### DISCUSSION TITLE
Activity disappear

### DISCUSSION THREAD

**Discussion by Kusieqs** [community/community#114531] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My activity disappeared for most months (April - September 2023) and I would like to ask if anyone knows why this happened? The only thing I can add is that the committs changed the owner to my natural name and not to my nickname. Recently I had a keylogger virus and it also appeared on github, but I don't know what happened.

**Comment by benogoulart** [comment]
Hi,

There are a few possible reasons why your contribution activity from that period is no longer showing:

- If the commits were rewritten (for example, by changing the author name or email), GitHub may no longer associate those commits with your account.
- If the email address used in the commits is no longer verified or linked to your GitHub account, those contributions may disappear from your graph.
- If the repositories became private, were deleted, or their default branch changed, your contribution history may also be affected.
- If your account was compromised due to the malware you mentioned, it's possible that changes were made to your repositories or commit history.

A few things you can check:

1. Verify that the email address used in those commits is still added and verified in your GitHub account.
2. Inspect one of the affected commits to see what **Author** name and email are recorded.
3. Check whether the repositories and their default branches are still the same as they were when the commits were made.
4. Review your account's **Security log** to see if there was any suspicious activity around the time your account may have been compromised.

Could you clarify:
- Are the commits themselves still present in the repositories, but just missing from your contribution graph?
- Or have the commits disappeared from the repositories entirely?
